hazelcast/src/test/java/com/hazelcast/internal/partition/MigrationCorrectnessTest.java

```diff
@@ -36,6 +38,9 @@
 // related issue https://github.com/hazelcast/hazelcast/issues/5444
 public class MigrationCorrectnessTest extends AbstractMigrationCorrectnessTest {
 
+    @ClassRule
+    public static ChangeLoggingRule changeLoggingRule = new ChangeLoggingRule("log4j2-debug-migrations.xml");
```

Comment:
You can change the name of this file from `log4j2-debug-migrations.xml` to `log4j2-trace-migrations.xml`